sdk/package/lib/mainFn/Daemon.ts

```diff
@@ -50,9 +50,6 @@ export class Daemon<
           exec,
         )
       const res = new Daemon(subc, startCommand)
-      effects.onLeaveContext(() => {
```

Comment:
removed here and in Daemons.build(). Use only 'term' and 'stop' to explicitly stop daemons.

the calls to 'stop' were already handled in RpcListener, SystemForStartOs and MainLoop